docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -3002,6 +2903,28 @@ Click on save and restart radius (pfcmd service radiusd restart).
 
 Now when a device connect with the username bob@acme.com then the authentication and accounting requests will be forwarded to one of the ACME RADIUS servers.
 
+RADIUS EAP Profiles
+~~~~~~~~~~~~~~~~~~~
+
+RADIUS EAP Profiles allow you to select a specific eap profile in FreeRADIUS based on the Realm of the user.
+
+In this eap profile you can define:
+ Certificates configuration.
+ OCSP configuration
+ EAP-Fast configuration
+ TLS Configuration
+
+And link all these configuration together.
+
+Per example the Realm ACME.COM needs to use the CA certificate from ACME CA and the other Realm needs to use the default one.
```

Comment:
Per example -> For example
Realm -> realm
the other Realm needs -> the other realms need